services/minespace-web/src/components/dashboard/mine/projects/Projects.tsx

```diff
@@ -18,6 +19,7 @@ export const Projects: FC = () => {
   const projects = useSelector(getProjects);
 
   const handleFetchData = () => {
+    dispatch(fetchPermits(mine.mine_guid));
     dispatch(fetchProjectsByMine({ mineGuid: mine.mine_guid })).then(() => {
```

Comment:
These two fetches look like they want to be in a Promise.all so that it gets set as loaded after both are completed.